liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerConfigDocument.java

```diff
@@ -0,0 +1,238 @@
+            configDropins = new File(getServerFile().getParent(), "configDropins");
+        }
+        
+        if (configDropins != null && configDropins.exists()) {
+            File overrides = new File(configDropins, "overrides");
+            
+            if (overrides.exists()) {
+                File[] cfgFiles = overrides.listFiles();
+                
+                for (int i = 0; i < cfgFiles.length; i++) {
+                    if (cfgFiles[i].isFile()) {
+                        parseDropinsFiles(cfgFiles[i]);
+                    }
+                }
+            }
+            
+            File defaults = new File(configDropins, "defaults");
+            if (defaults.exists()) {
+                File[] cfgFiles = defaults.listFiles();
+                
+                for (int i = 0; i < cfgFiles.length; i++) {
+                    if (cfgFiles[i].isFile()) {
+                        parseDropinsFiles(cfgFiles[i]);
+                    }
+                }
+            }
+        }
+    }
+    
+    private static void parseDropinsFiles(File file) throws Exception {
+        // get input XML Document 
+        Document doc = getDocumentBuilder().parse(new FileInputStream(file));
+        
+        parseApplication(doc, "/server/application");
+        parseApplication(doc, "/server/webApplication");
+        parseInclude(doc, "/server/include");
+    }
+    
+    private static Document getIncludeDoc(String loc) throws Exception {
+    
+        Document doc = null;
+        File locFile = null;
+        
+        if (loc.startsWith("http:") || loc.startsWith("https:")) {
+            if (isValidURL(loc)) {
+                URL url = new URL(loc);
+                URLConnection connection = url.openConnection();
+                doc = getDocumentBuilder().parse(connection.getInputStream());
+            }
+        }
+        else if (loc.startsWith("file:")) {
+           if (isValidURL(loc)) {
+               locFile = new File(loc);
+               if (locFile.exists()) {
+                   InputStream inputStream = new FileInputStream(locFile.getCanonicalPath());
+                   doc = getDocumentBuilder().parse(inputStream);    
+               }
+           }
+       }
+       else if (loc.startsWith("ftp:")) {
+           // TODO handle ftp protocol
+       }
+       // relative file path
+       else {
+           // check configDirectory first if exists
+           if (configDirectory != null && configDirectory.exists()) {
+               locFile = new File(configDirectory, loc);
+               if (locFile.exists()) {
+                   InputStream inputStream = new FileInputStream(locFile.getCanonicalPath());
+                   doc = getDocumentBuilder().parse(inputStream);    
+               }
+           }
+           
+           if (locFile == null || !locFile.exists()) {
+               locFile = new File(getServerFile().getParentFile(), loc);
```

Comment:
will this a problem if loc is an absolute path. i.e. `<include location"/tmp/serverInclude.xml" />`?